src/NServiceBus.Persistence.CosmosDB.PersistenceTests/PersistenceTestsConfiguration.cs

```diff
@@ -85,7 +85,7 @@ public Task Configure(CancellationToken cancellationToken = default)
             var partitionKeyPath = new PartitionKeyPath(SetupFixture.PartitionPathKey);
             var resolver = new ContainerHolderResolver(this, new ContainerInformation(SetupFixture.ContainerName, partitionKeyPath), SetupFixture.DatabaseName);
             SagaStorage = new SagaPersister(serializer, sagaPersistenceConfiguration);
-            OutboxStorage = new OutboxPersister(resolver, serializer, OutboxTimeToLiveInSeconds);
+            OutboxStorage = new LogicalOutboxBehavior(resolver, serializer, OutboxTimeToLiveInSeconds);
```

Comment:
This seems off. The Outbox persister implementation should still be called OutboxPersister